Xamarin.Forms.Core/Page.cs

```diff
@@ -437,6 +443,12 @@ public void SendAppearing()
 			var pageContainer = this as IPageContainer<Page>;
 			pageContainer?.CurrentPage?.SendAppearing();
 
+			if(this is MasterDetailPage mdp)
+			{
+				mdp.Master?.SendAppearing();
```

Comment:
This is the same logic used inside the native android MDP it should just be moved xplat for consistency